docs/extend-graphql-schema.md

```diff
@@ -3,4 +3,568 @@ id: extend-graphql-schema
+  // architecture when it is not relevant
+
+-  StoreInformation
++  StoreInformation,
++  PhysicalStores
+];
+```
+
+Congratulations! You now have a new custom module ready to enhance the data
+exposed in the GraphQL middleware.
+
+## Extend the GraphQL schema
+
+All the wiring is now done and it is time to develop the features of this
+module. In GraphQL, a good practice is to start thinking about the schema first
+from a business domain standpoint.
+
+It is important to name things with a language shared by the team and prevent
+exposing implementation details (ids, different names…) as much as possible. We
+recommend the reading of the Graphql documentation page
+[Thinking in Graphs](https://graphql.github.io/learn/thinking-in-graphs/).
+
+Let’s add the code to expose physical stores in our graph.
+
+### Define the schema
+
+Front-Commerce contains all the tooling to allow you to describe your schema
+using the expressive
+[GraphQL Schema Definition Language (SDL)](https://www.prisma.io/blog/graphql-sdl-schema-definition-language-6755bcb9ce51/).
+
+Create a `src/server/modules/physical-stores/schema.gql` file to contain the
+schema matching our use case. It could look like so:
+
+```gql
+type Query {
+  "Our physical stores"
+  physicalStores: [PhysicalStore]
+}
+
+"General information about a physical store"
+type PhysicalStore {
+  address: PostalAddress
+  openingHours: [OpeningHours]
+}
+
+"Postal address of a physical location"
+type PostalAddress {
+  street: String
+  city: String
+  postcode: String
+  country: String
+}
+
+"Opening hours for a given day"
+type OpeningHours {
+  day: DayOfWeek
+  openingTimes: [TimePeriod]
+}
+
+"Period of time within a day"
+type TimePeriod {
+  from: Time!
+  to: Time!
+}
+
+"24h time definition"
+type Time {
+  h: Int!
+  m: Int!
+}
+
+enum DayOfWeek {
+  MONDAY
+  TUESDAY
+  WEDNESDAY
+  THURSDAY
+  FRIDAY
+  SATURDAY
+  SUNDAY
+}
+```
+
+Then expose the schema using the `typeDefs` key of the module definition. Update
+the `src/server/modules/physical-stores/index.js` entrypoint:
+
+```diff
++import typeDefs from "./schema.gql";
++
+export default {
+-  namespace: "PhysicalStores"
++  namespace: "PhysicalStores",
++  typeDefs: typeDefs
+};
+```
+
+> Webpack is in fact taking care of converting SDL into a Javascript object
+> using an appropriate loader.
+
+**Congratulations again!** You should now be able to see these new types and
+query them in GraphQL.
+
+Try to execute the query below in Graph*i*QL:
+
+```
+{
+  physicalStores {
+    address {
+      street
+      city
+      country
+      postcode
+    }
+    coordinates {
+      latitude
+      longitude
+    }
+    openingHours {
+      day
+      openingTimes {
+        from {
+          h
+          m
+        }
+        to {
+          h
+          m
+        }
+      }
+    }
+  }
+}
+```
+
+Our GraphQL module does not yet have any code for sending content, so you must
```

Comment:
I would mention the reason of `null` after displaying the response.

As a user I don't understand an explanation if I don't already know what it's explaining.